main.go

```diff
@@ -167,7 +166,9 @@ Advanced options:
                            locking and execution commands.
                            [default: 16].
   --no-preserve-uid       Do not preserve UIDs for transferred files.
```

Comment:
We could also export this to files with _windows.go and _unix.go suffix. What do you think? Is it necessary?

Reply:
Oh, yeah, totally right there :)